entrypoint.sh

```diff
@@ -122,6 +118,23 @@ else
 fi
 echo ::endgroup::
 
+echo ::group::Code check
+# only run `make codecheck` if the Makefile has a `codecheck` target
+# (default to tools/code_check.sh otherwise)
+if grep -iq codecheck Makefile; then
+  make codecheck 2>&1
+
+  # don't continue if the code checker doesn't pass
```

Comment:
I think this is not necessary. Note the `set -e` in the beginning of this file. If the return code is non-zero, the script should exit.

```
$ set --help | grep "\-e"
      -e  Exit immediately if a command exits with a non-zero status.
```

Reply:
Good catch! I didn't notice the `set -e` at the beginning. I removed this exit code check in https://github.com/ignition-tooling/action-ignition-ci/pull/26/commits/a997193a8278dabe4bc20cfe2ef111ec438d2d1f and re-ran a few tests. All of them behaved as expected:

1. Default branch of `ign-tools` (`ign-tools1`) should pass GitHub actions using `tools/code_check.sh`: https://github.com/ignitionrobotics/ign-tools/runs/1558105250
2. Default branch of `ign-math` (`ign-math6`) should pass GitHub actions using `make codecheck`: https://github.com/ignitionrobotics/ign-math/runs/1558104303
3. Default branch of `ign-math` should fail GitHub actions using `make codecheck` by adding a really long comment that doesn't adhere to the 80 character width limit: https://github.com/ignitionrobotics/ign-math/runs/1558105007